controllers/asteroids

```diff
@@ -1,5 +1,6 @@
 const getAsteroidsWithinPeriod = require('../services/asteroidsData');
```

Comment:
please create `app` or `src` folder  and transfer all your folders to it

Reply:
Should the entry point file server.js be moved to this folder too?

Reply:
Thank you. src folder is created and all folders moved to it.

Reply:
yes please

Reply:
Done.